packages/world/test/World.t.sol

```diff
@@ -250,7 +251,9 @@ contract WorldTest is Test, GasReporter {
       coreRegistrationSystem.registerFunctionSelector.selector,
       coreRegistrationSystem.registerRootFunctionSelector.selector,
       coreRegistrationSystem.registerDelegation.selector,
-      coreRegistrationSystem.registerNamespaceDelegation.selector
+      coreRegistrationSystem.unregisterDelegation.selector,
```

Comment:
oof, we didn't have this before, eh?

Reply:
another reason for https://github.com/latticexyz/mud/issues/2155

Reply:
Yup, this was forgotten. Although this is a test file so whatever automation we add needs to apply here too.